frontend/components/molecules/Form/FieldSection.js

```diff
@@ -27,15 +27,36 @@ const DescriptionSection = ({ text }) => {
 const FieldSection = ({
     title,
     description = '',
-    type = 'text',
+    type,
     required,
     onChange,
     name,
     value,
     hasError = false,
-    errorText = 'Incorrect answer provided. Please double-check your answer.',
+    errorText,
     options = [],
 }) => {
+    // checks for errorText first if its undefined. If it is, then figure out the type and set the errorMessage based off the type. If errorText is defined
+    // then no need to check for type, it has a specific message from backend
+    if (typeof errorText === 'undefined') {
+        switch (type) {
+            case 'email':
+                errorText = 'Please enter a valid email.';
+                break;
+            case 'phone':
+                errorText = 'Please enter a valid 10 digit phone number.';
+                break;
+            case 'text':
+                errorText = 'Please provide a response to this question.';
+                break;
+            case 'radio':
+                errorText = 'Please select an option above.';
+                break;
+            case 'boolean':
```

Comment:
@victorr-su , I think you're missing some types. These are all the data types here:

![image](https://user-images.githubusercontent.com/19509993/222936490-837dfc9d-c8a1-4285-b3e4-669eb4899b5f.png)